pddl/parser/common.lark

```diff
@@ -58,7 +58,7 @@ DECREASE: "decrease"
 MAXIMIZE: "maximize"
 MINIMIZE: "minimize"
 TOTAL_COST: "total-cost"
-NUMBER: /[0-9]+(\.[0-9]+)*/
+NUMBER: /[-]?[0-9]+(\.[0-9]+)*/
```

Comment:
I think this should be left out of this PR (keep it to just disjunctive goals), and we'll tackle the single negative on that PR needing resurrection.